Makefile

```diff
@@ -95,7 +95,10 @@ bundle:
 
 setup: build db_create db_migrate
 
-publish: build
+migrate_if_needed:
+	bash -c 'if docker-compose run ${RAILS_CONTAINER} rake db:migrate:status | grep "^\s*down"; then docker-compose run ${RAILS_CONTAINER} rake db:migrate; fi'
+
+publish: build migrate_if_needed
 	bin/publish
```

Comment:
🤔 I'm not super familiar with the deployed production environment. Looking at the `bin/publish` script has me thinking that this wiring up of a `migrate_if_needed` action prior to publish would have the migrations occur in an environment that is _not_ production, but rather in a development or CI environment that is doing Docker image tagging and kubernetes service poking.

Reply:
Gotcha. I thought maybe the docker container that build was run on would then be promoted to production.